web/src/app/schedules/temp-sched/TempSchedDialog.tsx

```diff
@@ -82,6 +83,25 @@ export default function TempSchedDialog({
     },
   })
 
+  const [isAllowingNoCoverage, setIsAllowingNoCoverage] = useState(false)
+  const [isShowingNoCoverageWarning, setIsShowingNoCoverageWarning] =
```

Comment:
What are your thoughts on using names like `allowNoCoverage, setAllowNoCoverage` along with `hasCoverage, setHasCoverage`. Removes the double neg for what's the current coverage is to try and help

Reply:
Removed double negative from naming